### PR TITLE
Remove deprecated blend attribute from draw_aaline() & draw_aalines()

### LINKED ISSUE #1233: in pygame 2.2 remove draw.aaline is called with blend=False (2405)
#  **Issue &numero;2405 opened by *[illume](https://github.com/illume)* at 2020-12-19 15:26:57**

draw.aaline is called with blend=False

See PR https://github.com/pygame/pygame/pull/2261<hr>

#  Comments

# # #  **[zoldalma999](https://github.com/zoldalma999)* commented at 2020-12-19 15:36:19*

Also `mouse.set_system_cursor`

See PR # 2397

### PR DESCRIPTION
**Pull request by MyreMylar**
Should have removed this in 2.2.0 but it got forgotten in the shuffle. It has been deprecated for some time now, this just finally pulls it all the way out

fixes #1233

### CODE REVIEW

File: test/draw_test.py [hunk 1]
```diff
@@ -1072,7 +1072,7 @@ def _create_surfaces():
         for size in ((49, 49), (50, 50)):
             for depth in (8, 16, 24, 32):
                 for flags in (0, SRCALPHA):
-                    surface = pygame.display.set_mode(size, flags, depth)
```
Comment (MyreMylar) [inline test/draw_test.py:1075]: This is just a small unrelated change related to another deprecation that popped up while testing.

**Review by ankith26 — APPROVED** [review]
LGTM, thanks for the PR :partying_face:

**Review by Temmie3754 — APPROVED** [review]
Can confirm this resolves the depreciation. 👍
